.travis.yml

```diff
@@ -38,12 +42,15 @@ jobs:
     # OSX build removed due to long build startup delays
     # Restore matrix job entry from d2d67fab to test OSX
     - python: 2.7
+      <<: *disable_global
       stage: unsupported
       script: .misc/check_unsupported.sh
     - python: 3.3
+      <<: *disable_global
       stage: unsupported
       script: .misc/check_unsupported.sh
     - python: 3.4.3
+      <<: *disable_global
       stage: unsupported
       script: .misc/check_unsupported.sh
     - python: 3.6
```

Comment:
To make this section more intuitive and readable, can we consider merging `disable_global` in this section too and then overriding install?

Reply:
It is also partially possible to do this in https://github.com/coala/coala-bears/blob/master/.travis.yml#L62.

Reply:
Feel free to create a newcomer issue about the bears one; explain it fairly clearly, so domain knowledge isnt needed - only expect yaml skills to complete the task.